### PR TITLE
enhance core stat ut

### PR DESCRIPTION
**Pull request by sanxun0325**
### Describe what this PR does / why we need it
enhance core stat ut

### Describe how to verify it
go test ./...

### Special notes for reviews

### CODE REVIEW

File: core/stat/base/sliding_window_metric_test.go [hunk 1]
```diff
@@ -3,6 +3,8 @@ package base
 import (
 	"testing"
 
+	"github.com/alibaba/sentinel-golang/util"
```
Comment (louyuting) [inline core/stat/base/sliding_window_metric_test.go:6]: rearrange imports by "go imports"

File: core/stat/base/leap_array_test.go [hunk 2]
```diff
@@ -6,6 +6,8 @@ import (
 	"testing"
 	"unsafe"
 
+	"github.com/stretchr/testify/assert"
```
Comment (louyuting) [inline core/stat/base/leap_array_test.go:9]: rearrange imports by "go imports"

Comment (sanxun0325) [reply core/stat/base/leap_array_test.go]: updated

File: core/stat/base/sliding_window_metric_test.go (hunk 1 above)
Comment (sanxun0325) [reply core/stat/base/sliding_window_metric_test.go]: updated

**Review by louyuting — APPROVED** [review]
Thanks for contribution

**Pull request merged**